docs/src/examples/components/TextArea/Types/TextAreaFluidExample.shorthand.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react'
+import { TextArea } from '@stardust-ui/react'
+
+const TextAreaExample = () => <TextArea fluid placeholder="Type here..." />
+
+export default TextAreaExample
```

Comment:
```suggestion
export default TextAreaExampleFluid
```

File should be also renamed